pkg/build/graph/analysis/bc.go

```diff
@@ -162,6 +224,43 @@ func FindPendingTags(g osgraph.Graph) []osgraph.Marker {
 	return markers
 }
 
+// validImageStreamImage will cycle through the imageStream.Status.Tags.[]TagEvent.DockerImageReference and  determines whether an image with the hexadecimal image id
+// associated with an ImageStreamImage reference in fact exists in a given ImageStream
+func validImageStreamImage(tagNode *imagegraph.ImageStreamImageNode, imageStream *imageapi.ImageStream) (bool, string, string) {
```

Comment:
part of next push; also had another instance of old "tagNode" var name where "imageNode" was more appropriate